apps/guardian/src-tauri/tauri.conf.json

```diff
@@ -7,7 +7,7 @@
     },
     "package": {
         "productName": "Concordium Election Guardian",
-        "version": "0.1.3"
+        "version": "0.1.2"
```

Comment:
0.1.3 has not been released, so this change is to be consistent with how we release (i.e. bump version + align changelog in a PR)